### ISSUE TITLE
[Bug]: Click event is not fired from notifications event listeners when waking the application with notification.

### ISSUE THREAD

**Issue by onuronat001** [OneSignal/react-native-onesignal#1547]
### What happened?

In previous versions, there was an event listener called onOpened.

I am currently using the latest version, "react-native-onesignal": "^5.0.0". Among the event listeners, there are two events: "click" and "foregroundWillDisplay". "foregroundWillDisplay" works correctly when the application is open, but the "click" event works when the application is in the background or foreground and I click on incoming notifications. However, it does not work when the application is completely closed and is awakened by a notification. In this way, we cannot perform the desired actions with notifications when the application is closed. When I send notifications with content_available = true, it works by forcibly waking up the application, but then it tries to perform some tasks before the application is fully opened, such as resetting the badge count, for example.

### Steps to reproduce?

```Markdown
1. The application is completely closed.
2. A notification is sent.
3. The application is awakened by the notification.
4. You do not receive any response from any of the OneSignal listeners.
```

### What did you expect to happen?

I think the "click" event should be triggered, which I think replaces the "onOpened" event

### React Native OneSignal SDK version

Release 5.0.0

### Which platform(s) are affected?

- [X] iOS
- [ ] Android

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jennantilla** [comment]
Thank you for reporting this, @onuronat001! We're investigating and will get back to you as soon as possible.

**Comment by liamong0422** [comment]
Hi @jennantilla , any fix to this? I'm facing the same issue as well

**Comment by jennantilla** [comment]
@liamong0422 thank you for your patience!

We have a fix for this in our latest release, [5.0.1](https://github.com/OneSignal/react-native-onesignal/releases/tag/5.0.1). Please let us know if you have any additional questions or concerns!
